src/dune_lang/syntax.ml

```diff
@@ -203,27 +203,40 @@ module Error = struct
           ]
       :: repl )
 
-  let disabled loc t ~dune_lang_ver ~what =
-    let min_lang_version, min_dune_version =
-      let major, major_map =
-        Option.value_exn (Int.Map.min_binding t.supported_versions)
-      in
-      let minor, lang = Option.value_exn (Int.Map.min_binding major_map) in
-      ((major, minor), lang)
-    in
+  let inactive loc t ~dune_lang_ver ~what =
     User_error.raise ~loc
-      [ Pp.textf
-          "%s is available only when %s is enabled in the dune-project file. \
-           It cannot be enabled automatically because the currently selected \
-           version of dune (%s) does not support this plugin.\n\
-           You must enable it using (using %s ..) in your dune-project file. \
-           The first version of this plugin %s was introduced in dune %s."
-          what t.name
-          (Version.to_string dune_lang_ver)
-          t.name
-          (Version.to_string min_lang_version)
-          (Version.to_string min_dune_version)
-      ]
+      ( [ Pp.textf
+            "%s is available only when %s is enabled in the dune-project file. \
+             You must enable it using (using %s ..) in your dune-project file."
```

Comment:
Maybe we could print the maximum version available for the plugin (with that dune-lang version) instead of `..` ?

Reply:
Sounds good, I did that. And also added a disclaimer when the extension is experimental, to not make it too easy for users to rely on something experimental.